src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectSystem/SnapshotResolver.cs

```diff
@@ -29,6 +33,11 @@ public SnapshotResolver(IProjectSnapshotManager projectManager, ILoggerFactory l
         var miscellaneousProjectPath = Path.Combine(TempDirectory.Instance.DirectoryPath, "__MISC_RAZOR_PROJECT__");
         var normalizedPath = FilePathNormalizer.Normalize(miscellaneousProjectPath);
         MiscellaneousHostProject = new HostProject(normalizedPath, normalizedPath, FallbackRazorConfiguration.Latest, rootNamespace: null, "Miscellaneous Files");
+
+        _initializeTask = _projectManager.UpdateAsync(
```

Comment:
Why don't we move the Misc project into the project manager and add it on creation?

Reply:
I guess in a larger sense, does anything need to be notified that the misc project is added? Is that the issue?